UE5: Couldn't build on linux as PlatformFilemanager.h was renamed

Hi

I've added the plugin into my project but it can't be compiled:

```
Plugins/LuaMachine/Source/LuaMachine/Private/LuaBlueprintFunctionLibrary.cpp:15:10: fatal error: 'HAL/PlatformFilemanager.h' file not found
#include "HAL/PlatformFilemanager.h"
```

I've found that the file has a new name in UE5: `HAL/PlatformFileManager.h`. I suppose building projects is not affected on Windows as its filesystem is case insensitive.

Here is a possible fix. I couldn't make a pull request myself.

```c++
#if !defined(ENGINE_MAJOR_VERSION) || ENGINE_MAJOR_VERSION < 5
#include "HAL/PlatformFilemanager.h"  // UE4
#else
#include "HAL/PlatformFileManager.h"  // UE5
#endif
```

It works with UE5 but I haven't tested with UE4 (maybe there is no need in `!defined(ENGINE_MAJOR_VERSION)` part).

Could someone add this? Thanks

--- a/Source/LuaMachine/Private/LuaBlueprintFunctionLibrary.cpp
+++ b/Source/LuaMachine/Private/LuaBlueprintFunctionLibrary.cpp
@@ -12,7 +12,11 @@
 #include "IImageWrapper.h"
 #include "IImageWrapperModule.h"
 #include "IPlatformFilePak.h"
+#if ENGINE_MAJOR_VERSION >= 5
+#include "HAL/PlatformFileManager.h" 
+#else
 #include "HAL/PlatformFilemanager.h"
+#endif
 #if ENGINE_MAJOR_VERSION >= 5 && ENGINE_MINOR_VERSION > 0
 #include "AssetRegistry/IAssetRegistry.h"
 #include "AssetRegistry/AssetRegistryModule.h"

